
CondaError: Run 'conda init' before 'conda activate'

Traceback (most recent call last):
  File "/w/340/murdock/irgan/train.py", line 15, in <module>
    opt = TrainOptions().parse()         # get training options
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/w/340/murdock/irgan/options/base_options.py", line 128, in parse
    opt = self.gather_options()
          ^^^^^^^^^^^^^^^^^^^^^
  File "/w/340/murdock/irgan/options/base_options.py", line 90, in gather_options
    model_option_setter = models.get_option_setter(model_name)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/w/340/murdock/irgan/models/__init__.py", line 50, in get_option_setter
    model_class = find_model_using_name(model_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/w/340/murdock/irgan/models/__init__.py", line 28, in find_model_using_name
    modellib = importlib.import_module(model_filename)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/w/246/murdock/miniconda3/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'models.IR-GAN_model'
srun: error: gpunode3: task 0: Exited with exit code 1
